alchemy/src/neon/branch.ts

```diff
@@ -0,0 +1,544 @@
+import type { Context } from "../context.ts";
+import { Resource } from "../resource.ts";
+import { handleApiError } from "./api-error.ts";
+import { createNeonApi, type NeonApiOptions, type NeonApi } from "./api.ts";
+
+/**
+ * Properties for creating or updating a Neon branch
+ */
+export interface NeonBranchProps extends NeonApiOptions {
+  /**
+   * The ID of the project where the branch will be created
+   */
+  project_id: string;
+
+  /**
+   * Name of the branch
+   */
+  name?: string;
+
+  /**
+   * ID of the parent branch to create this branch from
+   */
+  parent_id?: string;
+
+  /**
+   * Log Sequence Number (LSN) of the parent branch to branch from
+   */
+  parent_lsn?: string;
+
+  /**
+   * Timestamp of the parent branch to branch from
+   */
+  parent_timestamp?: string;
+
+  /**
+   * Whether to adopt an existing branch if it already exists
+   * @default false
+   */
+  adopt?: boolean;
+}
+
+/**
+ * Response structure for branch API operations
+ */
+export interface NeonBranchType {
```

Comment:
This type looks mostly redundant. Why does it need to be exported? Just inline it in the apis or at least move it to the bottom of the file